src/settings_dialog.js

```diff
@@ -498,6 +498,10 @@ resetDehydrateCacheLocation.addEventListener("click", function(){
     }
 });
 
+// Specifies launcher version that currently running
+const launcherVersion = document.getElementById("launcherVersion");
+launcherVersion.textContent = `Launcher Version: ${app.getVersion()}`;
```

Comment:
I think this should use the existing approach to get the current version: https://github.com/Revolutionary-Games/Thrive-Launcher/blob/b3eb612917e774538370ac7763b68cf2754b292b/src/check_launcher_version.js#L11